MediaBrowser.Providers/Plugins/AudioDb/Configuration/config.html

```diff
@@ -0,0 +1,57 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <title>AudioDB</title>
+</head>
+<body>
+    <div data-role="page" class="page type-interior pluginConfigurationPage configPage" data-require="emby-input,emby-button,emby-checkbox">
+        <div data-role="content">
+            <div class="content-primary">
+                <form class="configForm">
+                    <label class="checkboxContainer">
+                        <input is="emby-checkbox" type="checkbox" id="enable" />
+                        <span>Enable this provider for metadata searches on artists and albums.</span>
+                    </label>
+                    <label class="checkboxContainer">
+                        <input is="emby-checkbox" type="checkbox" id="replaceAlbumName" />
+                        <span>When an album is found during a metadata search, replace the name with the value on the server.</span>
+                    </label>
+                    <br />
+                    <div>
+                        <button is="emby-button" type="submit" class="raised button-submit block"><span>Save</span></button>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <script type="text/javascript">
+            var PluginConfig = {
+                pluginId: "a629c0da-fac5-4c7e-931a-7174223f14c8"
+            };
+
+            $('.configPage').on('pageshow', function () {
+                Dashboard.showLoadingMsg();
+                ApiClient.getPluginConfiguration(PluginConfig.pluginId).then(function (config) {
+                    $('#enable').checked(config.Enable);
+                    $('#replaceAlbumName').checked(config.ReplaceAlbumName);
+
+                    Dashboard.hideLoadingMsg();
```

Comment:
Should happen in error part of promise

Reply:
You're right, but I just copied the structure of this page from every other plugin. What really needs to happen is a completely different way to handle the config pages so we don't have to rely on this terrible idea.

Reply:
So it's done wrong everywhere, lol